code/game/machinery/medical_pod/autodoc.dm

```diff
@@ -36,10 +36,10 @@
 	else
 		icon_state = "autodoc_open"
 
-/obj/structure/machinery/medical_pod/autodoc/get_examine_text(mob/user)
-	. = ..()
-	if(ishuman(user))
-		. += SPAN_NOTICE("It has [stored_metal] metal available for limb replacements.")
+///obj/structure/machinery/medical_pod/autodoc/get_examine_text(mob/user)
+//	. = ..()
+//	if(ishuman(user))
+//		. += SPAN_NOTICE("It has [stored_metal] metal available for limb replacements.")
```

Comment:
we don't need to keep commented code - people can always look back in the git diff to restore  functionality